bin/build

```diff
@@ -8,7 +8,7 @@ set -euo pipefail
 # (env GOOS=darwin GOARCH=amd64 go build) of project binaries
 IMAGE_NAME=conjur-authn-k8s-client
 TAG=dev
-VERSION="$(short_version_tag)"
+VERSION="${TAG_NAME//"v"}"
```

Comment:
this should probably be conditional, so that it's TAG_NAME if the value is set, and short_version_name if not.

it's not critical, but it means in non-tag pipelines the RH image will have empty `version` and `release` labels, up to you if you change it.

Reply:
it was changed in the second commit, I think it is better to have it simple, use the short_version_tag